src/reorderable-repeat.js

```diff
@@ -213,17 +214,27 @@ export class ReorderableRepeat extends AbstractRepeater {
         this.intention = intention;
       })
     ];
+    if (typeof this.group === 'string') {
+      this.type = this.group;
+    } else if (this.group) {
+      // group is a BindingExpression
+      this.group.targetProperty = 'type',
+      this._typeBinding = this.group.createBinding(this);
+      this._typeBinding.bind(this.scope);
+    }
```

Comment:
@bigopon can you have a look of my bind and unbind? In case I missed something that created memory leak.

Reply:
It looks great 👍
Did you test with an interpolation reorderable-repeat group? It should still work, though just to be sure:
```html
reorderable-group="${group}-body">${s}</div>
```

Reply:
Good idea.